### PR TITLE
#4910 new input max min

### LINKED ISSUE #4910: Explorer: Sidebar Filter post-fix
- [ ]  `NeoSidebarItem`
- [ ] min & max price new input design
---
- [ ] thin icon pack (ref redesign: icons in wallet dialog redesign #4830)

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/4826#issuecomment-1415950526_
            
## Ref 
- #4906 
- #4474

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Partially Closes #4910
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-02-05 at 4 03 22 PM

### CODE REVIEW

File: components/explore/SidebarFilter.vue [hunk 1]
```diff
@@ -128,13 +129,32 @@ const replaceUrl = (queryCondition: { [key: string]: any }) => {
...
+
 .sticky {
   position: -webkit-sticky;
   position: sticky;
   top: 72px;
   height: calc(100vh - 72px);
 }
 </style>
+
+<style lang="scss">
```
Comment (Jarsen136) [inline components/explore/SidebarFilter.vue:145]: There are two style tags in this file. Let's keep only one.

File: components/explore/SidebarFilter.vue [hunk 2]
```diff
@@ -20,24 +20,25 @@
...
-          <b-field label="MAX" label-position="inside">
+            <img src="/arrow-right.svg" />
             <b-input
               v-model="range.max"
+              custom-class="input-sidebar"
               min="0"
               step="any"
               type="number"
-              :placeholder="$t('query.priceRange.maxPrice')"
+              placeholder="MAX"
```
Comment (Jarsen136) [inline components/explore/SidebarFilter.vue:39]: Why do this? Let's add a new translation key for it.

File: components/explore/SidebarFilter.vue [hunk 3]
```diff
@@ -20,24 +20,25 @@
...
-          <b-field label="MIN" label-position="inside">
+          <div class="is-flex input-container mb-4">
             <b-input
               v-model="range.min"
+              custom-class="input-sidebar"
               type="number"
               min="0"
               step="any"
-              :placeholder="$t('query.priceRange.minPrice')"
+              placeholder="MIN"
```
Comment (Jarsen136) [inline components/explore/SidebarFilter.vue:30]: Let's add a new translation key for it.

**Review by Jarsen136 — CHANGES_REQUESTED** [review]
Pls also change the color of icon in dark mode.

**Comment by yangwao** [comment]
dark mode miss arrow 👀
image

**Comment by yangwao** [comment]
Explorer input max min 
- change the color of icon in dark mode.
-  Let's add a new translation key for it. 
- https://github.com/kodadot/nft-gallery/pull/4924/files/272c608265fc9bddd9f77aa44ac184bd36bce352#diff-243018571f28605fceba0ba9db964d3c1a8b5e9a00152d26f383eb09bd5bd071R39
- https://github.com/kodadot/nft-gallery/pull/4924/files/272c608265fc9bddd9f77aa44ac184bd36bce352#diff-243018571f28605fceba0ba9db964d3c1a8b5e9a00152d26f383eb09bd5bd071R30

**Comment by yangwao** [comment]
pay 20 usd

continue in 
- #4930

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 36.35 USD/KSM ~ 0.55 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x4e1982ba3b81a3de510605a73701d51c97c2faf5e5ad32db33e782a249e8b058](https://kusama.subscan.io/extrinsic/0x4e1982ba3b81a3de510605a73701d51c97c2faf5e5ad32db33e782a249e8b058)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**